### PR TITLE
Small fix for metastore runtime installation.

### PR DESCRIPTION
**Pull request by yao531441**

### CODE REVIEW

File: python/cloudtik/runtime/metastore/scripts/install.sh [hunk 1]
```diff
@@ -76,6 +76,7 @@ function install_hive_metastore() {
           tar -zxf hive-standalone-metastore-${HIVE_VERSION}-bin.tar.gz && \
           mv apache-hive-metastore-${HIVE_VERSION}-bin hive-metastore && \
           rm hive-standalone-metastore-${HIVE_VERSION}-bin.tar.gz)
+      wget --no-check-certificate https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.38/mysql-connector-java-5.1.38.jar -P $METASTORE_HOME/lib/
```
Comment (jerrychenhf) [inline python/cloudtik/runtime/metastore/scripts/install.sh:79]: Please check other wget to as to the parameters for downloading.
We need to the download thing be clean designed at the console output.

File: python/cloudtik/runtime/metastore/scripts/configure.sh [hunk 2]
```diff
@@ -28,7 +28,7 @@ do
     shift
 done
 
-if [ not $IS_HEAD_NODE == "true" ]; then
+if [ $IS_HEAD_NODE != "true" ]; then
```
Comment (jerrychenhf) [inline python/cloudtik/runtime/metastore/scripts/configure.sh:31]: is "if [ not $IS_HEAD_NODE == "true" ]; then" working?

If not, we need to change also in install.sh for the same check.

Comment (yao531441) [reply python/cloudtik/runtime/metastore/scripts/configure.sh:31]: Use "if [ not $IS_HEAD_NODE == "true" ]"  will report error with "[: too many arguments" in our docker image, so I change it.

**Comment by jerrychenhf** [comment]
The code looks good to me.

**Pull request merged**